### ISSUE TITLE
Request: Export the default `logToConsole` logger

### ISSUE THREAD

**Issue by hiddenist** [dequelabs/axe-core-npm#802]
I'm looking into adding some UI when accessibility violations are found while using `@axe-core/react`, but I still want to have the default console logging running. This would be fairly straightforward if the `logToConsole` function was exported.

https://github.com/dequelabs/axe-core-npm/blob/acf6b0fcbae2dc7069a15b17871a64dde66191b9/packages/react/index.ts#L314

**Comment by michael-siek** [comment]
Hey @hiddenist, I will bring this up with the team and will update you soon!

**Comment by michael-siek** [comment]
Hey @hiddenist 
Please feel free to follow this PR https://github.com/dequelabs/axe-core-npm/pull/812 for exporting `logToConsole`

**Comment by padmavemulapati** [comment]
This is not actually an issue, but the way results rendered to console to specific requirement. i.e.,"Request: Export the default logToConsole logger" so that it cannt be considered as a feature, which can implement all over the packages.

It can be applicable only for the react-componants, those can be rendered to the browsers and browser's devtools console would be showing the logs for `New Issues`

Steps validated:

1. create a component 
```index.html
<!DOCTYPE html>
<html>
  <head> </head>
  <body>
    <div id="container"></div>
    <img src=""/>
    <style>
      * {
        padding: 0;
        margin: 0;
      }

      html {
        font: 14px normal Arial, sans-serif;
        color: #626771;
        background-color: #fff;
      }

      body {
        padding: 60px;
        text-align: center;
      }

      h1 {
        font-size: 18px;
        margin-bottom: 20px;
      }
    </style>
    <script src="dist/bundle.js"></script>
  </body>
</html>
```
2. configure the axe to the componant

```
const axeConf = {
  rules: [
    { id: 'heading-order', enabled: true },
    { id: 'label-title-only', enabled: true },
    { id: 'image-alt', enabled: true },
    { id: 'link-in-text-block', enabled: true },
    { id: 'region', enabled: true },
    { id: 'skip-link', enabled: true },
    { id: 'help-same-as-label', enabled: true }
  ]
};
```
3. Render the above react-componant via browser (here started the server on http://localhost:8081/) using 

```
if (process.env.NODE_ENV !== 'production') {
  axe(React, ReactDOM, 1000, axeConf);
  const axeResults = {} // get axe-results from somewhere

   axe.logToConsole(axeResults) // logs to the devtools console
}
```
Output: logToConsole() is logging the axeResults to browser's console(devtools)

![image](https://github.com/dequelabs/axe-core-npm/assets/47409579/2f5bdfbe-13a4-4a9c-9807-d8214b6703ca)


## Environment:
| **Label**                         | **Value** |
| -------------------------- | ------ |
| **Product**                | _axe-core-npm/react_|
|**Version** | _4.7.4-62fd241.0_ |
| **OS-Details**      | _MAC- Intel Core i7 - 11.6.8  & Windows11 _ |
|**BrowserDetails**|_Chrome Version 117.0.5938.88 (Official Build) (64-bit) & Firefox 117.0_|

**Comment by padmavemulapati** [comment]
As per @michael-siek , this is the request only for specfic requirement, may not be feature that can implement for all other packages in npm
